### PR TITLE
Feature: Update application fields for new due diligence requirements

### PR DESCRIPTION
**Pull request by stephatmoves**
Occupation and nationality are required fields. Source of income and annual income are only required for non-US nationals. Unit hasn't given a timeline for when they'll make these changes, but the requirements will be in force by the end of May.

### CODE REVIEW

File: tests/applications.spec.ts [hunk 1]
```diff
@@ -67,6 +67,8 @@ describe("Applications", () => {
                     "countryCode": "1",
                     "number": "1555555578"
                 },
+                "nationality": "US",
```
Comment (dev-dennis-guye) [inline tests/applications.spec.ts]: Can you remove the update to this test? If there are compilation issues resolve them using type assertions. Reason is that this is a test that runs on Unit's CI.

File: tests/testHelpers.ts [hunk 2]
```diff
@@ -41,8 +45,10 @@ export function createIndividualApplicationWithSelfieVerification(unit: Unit) {
             fullName: createFullName("Richard", "Hendricks"),
             dateOfBirth: "2001-08-10",
             address: createAddress("20 Ingram St", null, "Forest Hills", "CA", "11375", "US"),
+            nationality: "US",
             email: "april@baxter.com",
             phone: createPhone("1", "5555555555"),
+            occupation: "GigWorker",
```
Comment (dev-dennis-guye) [inline tests/testHelpers.ts]: This is a valid occupation. 

https://docs.unit.co/types#occupation

Comment (stephatmoves) [reply tests/testHelpers.ts]: Yes that's right, the enum is pulled from the doc. Sorry could you clarify what needs to be changed here with the value?

File: tests/applications.spec.ts (hunk 1 above)
Comment (stephatmoves) [reply tests/applications.spec.ts]: Gotcha!

File: tests/testHelpers.ts (hunk 2 above)
Comment (dev-dennis-guye) [reply tests/testHelpers.ts]: There is nothing wrong. I usually leave these comments to signal to the next reviewer that a key change is OK.

**Review by dev-dennis-guye — APPROVED** [review]
LGTM - merge the change onto the fork.

**Comment by dev-dennis-guye** [comment]
@stephatmoves another contributor has added the [change](https://github.com/unit-finance/unit-node-sdk/pull/330). I'll create a new release of unit-node-sdk e to include this change.

**Pull request closed without merging**